src/AndcultureCode.CSharp.Data.SqlServer/Migrations/FlattenedMigration.cs

```diff
@@ -0,0 +1,100 @@
+using Microsoft.EntityFrameworkCore;
```

Comment:
I'm good to approve, but do wonder what sort of integration test we might be able to make for this. Mind creating a github issue and linking it in a comment file as a class level comment?

Reply:
Yea, it's a little tricky because the core functionality that you'd want to test is the validate method, which is grabbing at the migration history table -- I'd think for an integration test we'd have to break that out into a method that could be mocked, as you likely wouldn't want to write test data with that table even on a test DB. 

Alternatively if this were in a full blown project (like the Boilerplate) and you had a migration ID you could test with, one could assume that all migrations have been run on the test DB prior to test and then integration test that way.

Either way, I'll add a comment and create a github issue to be revisited later.